### ISSUE TITLE
Ensure that all print methods are consistent

### ISSUE THREAD

**Issue by kloppen** [ComtekAdvancedStructures/cmstatr#5]
- Spacing
- Capitalization
- General look and feel

**Comment by kloppen** [comment]
`ad_ksample` print method is missing a space after alpha:

```
Call:
ad_ksample(data = ., x = strength.norm, groups = batch)

N =  28 	k =  3 
ADK =  6.65 	p-value =  0.0025892 
Conclusion: Samples do not come from the same distribution (alpha= 0.025 )
```

**Comment by kloppen** [comment]
Fixed in commit 1b118fd

Any other issues with print methods will be tracked by separate issues.
